### ISSUE TITLE
Reset import give a 404 not found

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ravi5677/gisgraphy#2]
```
What steps will reproduce the problem?
1. launch the import
2. go to tha admin menu
3. click on reset import


it gives a 404 but should give the ability to reset import





```

Original issue reported on code.google.com by `davidmasclet@gmail.com` on 17 Nov 2009 at 9:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `davidmasclet@gmail.com` on 19 Nov 2009 at 6:36
- Added labels: **affects_version-2.0beta**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 2.0 beta2
```

Original comment by `davidmasclet@gmail.com` on 19 Feb 2010 at 9:30
- Changed state: **Fixed**
- Added labels: **fix_version-2.0beta2**
